src/logging.rs

```diff
@@ -14,10 +14,10 @@ use std::os::unix::net::UnixDatagram;
 use std::str::FromStr;
 
 /// Default path of the systemd-journald `AF_UNIX` datagram socket.
-pub static SD_JOURNAL_SOCK_PATH: &str = "/run/systemd/journal/socket";
+pub const SD_JOURNAL_SOCK_PATH: &str = "/run/systemd/journal/socket";
```

Comment:
Why this change?  Is that a good idea?  As far as I understand `const` it gets inlined into callers, which I don't think is a good thing to do for global environment definitions (what if this path changes), and wouldn't that constitute an ABI break?

Reply:
Basically it allows a broader set of usages and optimizations.

Rust doesn't have a stable ABI, but I don't think it would be considered an API break to change the value of a const variable.  The API is that it's a const &str.  You're guaranteeing that any value there will be created using a const value or function, which is a stronger guarantee than static.  All that said, I've reverted this.

Reply:
Ah, sorry you're right.   So perhaps this is a good change; I believe I don't know Rust so well as to say.  @lucab I think this is your call 🙂